src/stats.jl

```diff
@@ -149,6 +149,23 @@ function Base.delete!(o::CountMap, level)
     o.n -= x
     o
 end
+function Statistics.mean(cm::CountMap{T}) where T
+	cmdict = value(cm) 
+    k = sort(collect(keys(cmdict))) # Sorted keys (values)
+    f = [cmdict[x] for x in k] # Occupancies (frequency) of each values
+    weightedMean = sum(k .* f) / sum(f) # Compute the weighted mean
+    return weightedMean
+end    
+function Statistics.quantile(cm::CountMap{T}, q::Float64) where T
+	cmdict = value(cm) 
+    k = sort(collect(keys(cmdict))) # Sorted keys (values)
+    f = [cmdict[x] for x in k] # Occupancies (frequency) of each values
+    cum = cumsum(f) ./ sum(f) # Compute the cumulative frequency for each value
+    idx = findfirst(x -> x ≥ q, cum) # Return the first index where the cumulative frequency is greater than or equal to `q`
+	
+    return k[idx]
+end
+Statistics.quantile(cm::CountMap{T}, qvec::Vector{<:Float64}) where T = map(x -> Statistics.quantile(cm, x), qvec)
```

Comment:
There already exists a `quantile(v, w::AbstractWeights, p)` method from StatsBase.  We shouldn't re-implement it here, but rather use something like `v = collect(keys(cm)); w = fweights(collect(values(cm)))`